controllers/users-controller.js

```diff
@@ -26,13 +26,32 @@ exports.registerUser = (appReq, appRes) => {
   });
 
   /**
-   * TODO
-   * 1. validate to ensure null values are passed for name,
-   *   email, or password.
-   * 2. verify email is valid i.e. '....'@'...'.'...'
-   * 3. validate for secure passwords and no easy to guess passwords
+   *  Checks whether any of the fields are empty while submission.
```

Comment:
Please move the entire credentials validation logic to the beginning of the function. We should avoid doing any database related tasks unless we have valid user credentials.

Are you working in an IDE or a simple text editor? I ask because the current code is failing ESlint validations. If the latter, I can clean up on my end when I merge once it passes review else can you make sure you're running with correct depencies.